Modules/_winapi.c

```diff
@@ -3166,6 +3166,11 @@ static int winapi_exec(PyObject *m)
     #define COPY_FILE_REQUEST_COMPRESSED_TRAFFIC 0x10000000
 #endif
     WINAPI_CONSTANT(F_DWORD, COPY_FILE_REQUEST_COMPRESSED_TRAFFIC);
+#ifndef COPY_FILE_DIRECTORY
+    // Only defined in newer WinSDKs
```

Comment:
It's only defined if `NTDDI_WIN10_VB` is set, which is mainly controlled through `Py_NTDDI` in `PC/pyconfig.h`. I don't recall off the top of my head whether we can assume this version yet - we might be able to for 3.14.

If we can't set that as a minimum OS, we probably need a runtime version check on whether to define it (or certainly to use it), as `CopyFile2` is going to fail on unknown flags.

Reply:
If the flag isn't supported, `CopyFile2()` will fail with `ERROR_INVALID_PARAMETER` (87). The low-level `copyfile()` could return `NotImplemented` in this case. The `copy()` method could handle this by calling the fallback implementation in `PathBase.copy()`.

Reply:
Right, thanks. I think for now I'm going to remove support for `follow_symlinks=False` from this PR and do it in a follow-up PR instead, as it might be a can of worms.